packages/odyssey-react-mui/src/Typography.tsx

```diff
@@ -147,6 +152,7 @@ const Typography = ({
       tabIndex={-1}
       translate={translate}
       variant={typographyVariantMapping[variant]}
+      role={isPresentational ? "presentation" : undefined}
```

Comment:
Just to be sure, does passing `undefined` here result in using the default role or does it set the `role` attribute to a string `"undefined"`?